src/main/webapp/WEB-INF/views/login.jsp

```diff
@@ -0,0 +1,22 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: elena
+  Date: 25-Jan-21
+  Time: 18:23
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Login page</title>
+</head>
+<body>
+<h1>Login page</h1>
+<form action="${pageContext.request.contextPath}/login", method="post">
+    Driver login:<input type="text" name="login" pattern="^[a-z]*">
+    Driver password :<input type="password" name="pwd">
```

Comment:
what if I input nothing in here?

Reply:
we will have DataProcessingException with message "Can't find driver by login"

Reply:
but is it ok for user to see an exception stacktrace in browser? google `required`